support-modules/acquisition-events/src/main/scala/com/gu/support/acquisitions/bigquery/AcquisitionDataRowMapper.scala

```diff
@@ -29,29 +33,30 @@ object AcquisitionDataRowMapper {
       acquisition.componentId.map("component_id" -> _),
       acquisition.componentType.map("component_type" -> _),
       acquisition.source.map("source" -> _),
-      acquisition.campaignCode.map("campaign_codes" -> List(_).asJava),
+      acquisition.campaignCode.map((code: String) => "campaign_codes" -> new JSONArray(List(code).asJava)),
```

Comment:
Is there a reason why we have to specify the type of `code` here?

Reply:
I think it's fairly standard to leave lambda params untyped unless there is some particular reason to type them because  it reduces clutter but I'm not that fussed

Reply:
Sure I’ll take it out, I don’t have strong feelings about it either